src/brackets.js

```diff
@@ -425,7 +425,31 @@ define(function (require, exports, module) {
                 node = node.parentElement;
             }
         }, true);
-        
+
+        // on Windows, cancel every other scroll event (#10214)
+        // TODO: remove this hack when we upgrade CEF to a build with this bug fixed:
+        // https://bitbucket.org/chromiumembedded/cef/issue/1481
+        var winCancelWheelEvent = true;
```

Comment:
It's actually already like this - see the function call, where `winCancelWheelEvent` is first negated (so in the first call, it's now set to false) and then, we check whether it's truthy.
I don't know why I did it this way round (yeah, the other would probably make more sense), though.